scripts/fleet-install-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Check whether insights-agent is part of the changed charts
+cd /charts
+CHANGED="$(ct list-changed --config ./scripts/ct.yaml)"
+
+case "$CHANGED" in 
+  *insights-agent*)
+    echo "The changed charts include insights-agent. Running fleet install test"
+
+    # We use the be-main server to do the test. This is not ideal but will suffice for now
+    # Here we check to make sure that the server is up
+    URL="https://be-main.k8s.insights.fairwinds.com" 
+    retry=0
+    while ! curl -k --no-progress-meter $URL; do
+      echo "Server is not up yet. Waiting for it to start...";
+      sleep 20
+
+      retry=$(( $retry + 1 ))
+      if [ $retry -gt 30 ]; then
+        echo "Unable to curl the server. Giving up on it."
+        exit 1
+      fi
+    done
+
+    # Install insights-agent using the fleet install method
+    random_number=$((1 + $RANDOM % 100))
+    cluster_name="test-fleet-$random_number"
+    echo -e "\nServer is up. Running Fleet install for cluster $cluster_name"
+    
+    helm dependency build ./stable/insights-agent
+    helm upgrade --install insights-agent ./stable/insights-agent \
+      --namespace insights-agent \
+      --create-namespace \
+      --set insights.organization="acme-co" \
+      --set insights.cluster=$cluster_name \
+      --set insights.host=$URL \
+      --set fleetInstall=true  \
+      --set insights.tokenSecretName=insights-token \
+      --set insights.apiToken="thisisanadmintoken" \
```

Comment:
I expected these values to be set in `./stable/insights-agent/ci/fleet-install-values.yaml`. It'll probably be easier to maintain there